core/feconf_test.py

```
@@ -106,3 +108,13 @@ def test_terms_page_last_updated_is_in_sync_with_terms_page(self) -> None:
                 ).find_all('li')
             )
         self.assertEqual(feconf.TERMS_PAGE_LAST_UPDATED_UTC, max_date)
+
+    def test_prod_mode_frontend_dirs_match_angular_build_output(self) -> None:
+        with self.swap(constants, 'DEV_MODE', False):
```

Ruff (0.15.15)

[warning] Boolean positional value in function call

(FBT003)